prometheus_toolkit.md

```diff
@@ -0,0 +1,118 @@
+
+Some of the products strengths based on collection of quantifiable data would be the
+* Storing of data metric
+* Support of code queries
+* Providing data source to any [Grafana](https://grafana.com/) dashboard
+* Ease of deployment
+* The use of PromQL that is really good for visualisation purposes
+
+
+## Weaknesses
+
+When it comes to using Prometheus for in depth event logging you are better with using InfluxDB. There is several other systems that you would have a better use of when it comes to more specified tasks and processes.
+ie. 
+* Sensus is better for collecting and processing hybrid observability data(this includes metrics and/or events)
+* Graphite is better when you need/want a clustered solution that can hold historical data long term.
+
+
+Prometheus could be a little more userfriendly with the UI then it is. But once you get the hang of it, it's pretty easy. Learning PromQL can be tedious to do as well. This is Prometheus own query language that can be complex. 
+
+
+
+## Comparison
+
+Compared to [InfluxDB](https://www.influxdata.com/), Prometheus is a better choice if you are primarily doing metrics, it also has a more powerfull query language, alerting and notification functionality then InfluxDB has. Both of the systems has their strengths and weaknesses, where one is better then the other. 
+
+With Prometheus you get servers that are run independently of each other and only rely on their local storage for their core functionality: scraping, rule processing and alerting.
+
+The comercial InfluxDB offering is, by design, a distributed storage cluster with storage and queries being handled by many nodes at once. With that it means that the commercial version of InfluxDB will scale horizontally easier then Prometheus, but at the same time you need to manage higher complexity of a distributed storage systems strait of from the start. Prometheus is easier to setup and run. 
+
+There is an open source version of InfluxDB that is similar to Prometheus, but not completly the same when it comes to the architecture of the system.
+
+Both systems has a data model that has key-value pairs as labels, that is called tags. They both use support float64 data types but Prometheus offers this with limited support for strings and millisecond resolution timestamps, where InfluxDB offers has full support for it, along side of int64, and bool as well.
+
+If you primarily doing event logging then InfluxDB is more suitable for this type of monitoring.
+
+
+## Summary
+Prometheus is a great tool for systems monitoring and alerting, you can slice and dice the multi-dimensional data at will. Easier overview of samples that is stored. 
```

Comment:
I think this is a repeat of the introduction

Reply:
changed up the summary so that it is not a direct repeat of the introduction